tests/unit/misc/test_editor.py

```diff
@@ -249,7 +249,6 @@ def test_failing_unwatch(qtbot, caplog, monkeypatch):
     assert caplog.records[-1].msg == message
 
 
-
```

Comment:
This is an unrelated change, but was failing CI... probably introduced in master.